vite.config.ts

```diff
@@ -1,16 +1,19 @@
 /// <reference types="vitest" />
 
-import path from 'path';
+import * as dotenv from 'dotenv';
 import { defineConfig } from 'vite';
 import tsconfigPaths from 'vite-tsconfig-paths';
 
+dotenv.config();
+
+const pkg = process.env.npm_package_name;
+console.log('vitest:', pkg);
+
 export default defineConfig({
   plugins: [tsconfigPaths()],
   test: {
-    globalSetup: path.resolve(
-      __dirname,
-      './libs/model/src/tester/vitestDatabaseSetup.ts',
-    ),
+    sequence: { concurrent: false },
+    fileParallelism: false, // pkg !== '@hicommonwealth/model',
```

Comment:
If you add this here then you can remove `--fileParallelism=false` from all the commands